Glossary.md

```diff
@@ -0,0 +1,74 @@
+# Glossary of Terms
+
+## Domain
+
+A specific sphere of activity or knowledge. For example, cyber security, software, bills of material, licensing, software security, etc
+
+## Ontology
+
+An explicit specification of concepts, terms, categories, relationships, etc. for a particular scoped domain forming a language or basis for communication and analysis of information within the scoped domain. SPDX 3.0 is an ontology targeted to support the scoped domain of bills of material.
+
+## Class
+
+A represention of a scope/set of individual instances of a particular “concept” (e.g., File, Person, ExternalReference, etc.). Each individual instance of a class has an IRI ID and is asserted as a member of a particular class via a type statement.
+
+## Domain Class
+
+A class representing a concept of primary focus within a given domain. In a graph or serialized set of instance content this would be the granularity of what would be a node in the graph or a top-level object in the serialization set. This is typically similar to the scoping seen in labeled property graphs. In SPDX 3.0, all domain classes are subclasses of Element.
+
+## Non-domain Class
+
+A class representing a concept of secondary or tertiary focus within a given domain that is a characterization of some aspect of a domain class. When structuring the graph for a domain use case it is desirable and more practical for the domain graph nodes to only consist of domain class instances and object properties. Any non-domain classes and datatype properties would be considered internal adornments on the domain objects. In SPDX 3.0, all defined classes that are not subclasses of Element are non-domain classes.
```

Comment:
Same comment as above, we either need to make License a subclass of Element OR remove the statement that all domain classes are subclasses of Element.